app/server/Mint/types.ts

```diff
@@ -0,0 +1,14 @@
+export enum BitcoinTransactionStatus {
+	SUCCESS = "SUCCESS",
+	FAILED = "FAILED",
+	PENDING = "PENDING",
```

Comment:
the statuses we have are `broadcasted`, `confirming` <- here we have the number of confirmations we need to display nicely. `how many confirmations/ out of confirmation_threshold` eg. `(2/6)`, `mined` (success), `reorg` (failed), `failed` (failed)

Reply:
it is NOT done, the statuses are incorect and we need the number of confirmations